core/conn/jdbcT4/pom.xml

```diff
@@ -27,11 +27,34 @@
 -->
 
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.trafodion.jdbc.t4</groupId>
-  <artifactId>jdbcT4</artifactId>
+  <parent>
+    <groupId>org.apache.trafodion</groupId>
```

Comment:
This may not work as the package names does not contain apache in them. First we will need to make the package name change for T4